internal/oci/container.go

```diff
@@ -344,6 +344,24 @@ func (c *Container) exitFilePath() string {
 	return filepath.Join(c.dir, "exit")
 }
 
+// IsRunning is a function that checks if a specific pid is running
+// it is used to check a container state when we don't want (or don't trust) a `$runtime state` call
+func (c *Container) IsRunning() bool {
+	pid := c.state.Pid
+	process, err := os.FindProcess(pid)
```

Comment:
```suggestion
	process, err := os.FindProcess(c.state.Pid)
```

Reply:
because the function does not hold the lock, I am worried about the pid changing from underneath it. saving the value causes this function to run atomically

Reply:
though, I see now we don't even use this variable anywhere else. taking the change as suggested...

Reply:
fixed